chpl/chpl-service/src/main/java/gov/healthit/chpl/ChplCacheConfig.java

```diff
@@ -72,7 +72,6 @@ public CacheManager cacheManager() {
 
         backingManager.addCacheIfAbsent(createEternalCache(CacheNames.EDITIONS));
         backingManager.addCacheIfAbsent(createEternalCache(CacheNames.EDITION_NAMES));
-        backingManager.addCacheIfAbsent(createEternalCache(CacheNames.FIND_SURVEILLANCE_NONCONFORMITY_STATUS_TYPE));
```

Comment:
Not sure if you've done this but this means you can update the ehcache.xml files in STG and PROD environments (and gitlab)

Reply:
Added checkbox items to handle when we move to STG and PROD.